### ISSUE TITLE
Assassin's Creed: Origins Resource Management Tune-Up

### ISSUE THREAD

**Issue by palci12** [Kaldaien/SpecialK#15]
Hi. I saw this release and wanted to try it on AC origins right away but keyboard stopped responding and i was stuck at title screen. Is this mod compatible with origins? Or its just for odysey?
Thank you.

**Comment by Habitual-line-stepper** [comment]
I am also wondering if it is compatible with origins. Would be great.

**Comment by Idearum** [comment]
Yes, it is compatible with Origins. Have you tried disabling Uplay's overlay? I did not experience that issue right now when I tested it out.

**Comment by palci12** [comment]
Yes i disabled uplay overlay, but i have enabled geforce experience overlay. Maybe i should try do disable fulscreen optimization in windows too.

**Comment by Idearum** [comment]
It wouldn't hurt. You may also want to disable Ansel, as this has been known to cause compatibility issues: https://pcgamingwiki.com/wiki/Nvidia#Ansel

If you upload your modules.log and dxgi.log to pastebin.com and link them here I can take a look and see if I can spot any other possible compatibility issue.

**Comment by palci12** [comment]
I uninstaled gefroce experience and i dont have Ansel at all. I dont have dxgi.log or modules.log. Question is if i Installed it corretly.

**Comment by Idearum** [comment]
You seem to be using a pirated copy of AC: Origins. Compatibility with pirated copies is not guaranteed, nor supported.

For what it's worth, your logs also indicate that Ansel is being loaded.

**Comment by palci12** [comment]
I am ashamed for using pirated copy but even with disabled Ansel its not working. Well maybe its because of pirated copy. Anyway tahnks for your help.

**Comment by thehurtmaster** [comment]
It's definitely because of the pirated copy. Running this through a debugger shows that it validates files. It's not that. Kaldaein is known for being a cunt and putting DRM in mods.

**Comment by Idearum** [comment]
> It's definitely because of the pirated copy. Running this through a debugger shows that it validates files. It's not that. Kaldaein is known for being a cunt and putting DRM in mods.

The "DRM" you speak of is a simple Steam API DLL validation check which can easily be disabled and is a part of Special K's Steam enhancements. Said DLL validation check isn't even applicable in this case as the user's log indicates that this is a cracked Uplay copy and not a Steam copy, which wouldn't even have the DLL file in question. Special K does not include any Uplay related enhancements and therefore does not include a validation check for Uplay DLL files.

We basically have no idea what the issue is. It could be that it is a pirated copy, as those are not supported and nobody cares whether Special K works or not for those, or it could be another incompatibility issue altogether.

**Comment by fisherd80** [comment]
Hello,

I am trying to get SpecialK working with Assassin Creed Orgins.  Followed the advices posted online but still having problems.  Can load Origins but the keyboard is not working and the game crashes, the same issue that palci12 had.

I deactivated all overlays and Ancel in the settings, and still cannot get SK to work.  Any further advise would be welcome.
[crash.log](https://github.com/Kaldaien/SpecialK/files/4117281/crash.log)
[dxgi.log](https://github.com/Kaldaien/SpecialK/files/4117282/dxgi.log)
[modules.log](https://github.com/Kaldaien/SpecialK/files/4117283/modules.log)
